kodilla-exception/src/main/java/com/kodilla/exception/test/FirstChallenge.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.exception.test;
+
+public class FirstChallenge {
+    public double divide(double a, double b) throws ArithmeticException {
+
+        if (b == 0) {
+            throw  new ArithmeticException();
+        }
+
+        return a /b;
+    }
+
+    public static void main(String[] args) {
+        double result = 0;
+        FirstChallenge firstChallenge = new FirstChallenge();
+
+        try {
+            result = firstChallenge.divide(3, 0);
+            System.out.println(result);
+
+        } catch (ArithmeticException e) {
+            System.out.println("The divider must be larger than zero");
+        } finally {
+            System.out.println("Try again.");
```

Comment:
po co to? tutaj powinien byc rezultat wydrukowany

Reply:
Uznałem, że wynik tylko wtedy, gdy nie rzuca wyjątku. Poprawiłem